### PR TITLE
Converting all variable type definitions from var to const or let

### PR DESCRIPTION
**Pull request by pacostas**
Replacing var type definitions in documentation to const or let.

### CODE REVIEW

**Review by abstractj — DISMISSED** [review]

**Comment by abstractj** [comment]
@jonkoops LGTM, but I will wait for your approval before merge.

File: aggregation/navbar.html
Comment (jonkoops) [inline aggregation/navbar.html]: Why is `newtoc` defined with `let`? As far as I can see this variable is never re-assigned so should be fined to make it a constant.
```suggestion
    const newtoc = document.createElement('div');
```

File: authorization_services/topics/enforcer-js-adapter.adoc
Comment (jonkoops) [inline authorization_services/topics/enforcer-js-adapter.adoc]: Same here, this is never re-assigned so can also be constant.
```suggestion
const authorizationRequest = {};
```

File: authorization_services/topics/policy-js-policy.adoc
Comment (jonkoops) [inline authorization_services/topics/policy-js-policy.adoc]:
```suggestion
const permission = $evaluation.getPermission();
```

File: securing_apps/topics/oidc/javascript-adapter.adoc
Comment (jonkoops) [inline securing_apps/topics/oidc/javascript-adapter.adoc]:
```suggestion
    const req = new XMLHttpRequest();
```

**Review by jonkoops — CHANGES_REQUESTED** [review]

**Comment by jonkoops** [comment]
Looks like we have some failing tests. Can we get a rebase?

**Comment by pacostas** [comment]
Sure, just rebased it

**Review by jonkoops — APPROVED** [review]

**Comment by jonkoops** [comment]
The problems with external links are unrelated to the changes here. @abstractj could you go ahead and merge this PR?

**Review by abstractj — APPROVED** [review]

**Comment by jonkoops** [comment]
@pacostas Thanks for your contribution :100:

**Pull request merged**
